test/new-objectid.test.js

```diff
@@ -0,0 +1,81 @@
+// Copyright IBM Corp. 2013,2020. All Rights Reserved.
+// Node module: loopback-connector-mongodb
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+require('./init.js');
+
+let Book, Chapter;
+const ds = global.getDataSource();
+const objectIDLikeString = '7cd2ad46ffc580ba45d3cb1f';
+const objectIDLikeString2 = '7cd2ad46ffc580ba45d3cb1e';
+const promisify = require('bluebird').promisify;
+
+
+describe.only('New ObjectID', function() {
+
+  const Book = ds.createModel(
+    'Book',
+    {
+      oId: {type: String, mongodb: {dataType: 'objectID'}},
+      oIds: {type: [String], mongodb: {dataType: 'objectID'}},
+      title: String,
+      strId: String,
+      strIds: [String],
+      authorId: {type: [String], mongodb: {dataType: 'objectID'}}
+    }
+  );
+
+  const Author = ds.createModel(
+    'Author',
+    {
+      name: String
+    }
+  );
+
```

Comment:
Please add few tests to verify how nested properties are handled.

See the tests we added for Decimal128 in fb41e40777 and 25122e02a59 for inspiration.